go/consensus/cometbft/apps/vault/query.go

```diff
@@ -0,0 +1,67 @@
+package vault
```

Comment:
Could update txsource/workload/queries.go with the added queries.

Although these won't really test much unless there is also a vault workload added that populates the state. This should probably also be added, but can be done in a later PR.

Reply:
Yeah let's do this in a later PR.